### PR TITLE
chore(contributing): update contributor guidelines

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,116 @@
...
-For bug fixes that address open issues, the footer should be formatted like so:
+Provide the system information which is not limited to the below:
 
-```bash
-Closes #17, #26
-```
-and for Pull Requests, use the format:
+- OS: [e.g. macOS, Windows]
+- Browser (if applies) [e.g. chrome, safari]
+- Version of faux-cdn: [e.g. 5.0.0]
```
Comment (anescobar1991) [inline CONTRIBUTING.md]: This should be `jest-json-schema`.

File: README.md [hunk 2]
```diff
@@ -231,6 +231,8 @@ and to recipients of software distributed by American Express, You reserve all
 right, title, and interest, if any, in and to Your Contributions. Please [fill
 out the Agreement](https://cla-assistant.io/americanexpress/).
 
+Please feel free to open pull requests and see [CONTRIBUTING.md](./CONTRIBUTING.md) to learn how to get started contributing..
```
Comment (10xLaCroixDrinker) [inline README.md]:
```suggestion
Please feel free to open pull requests and see [CONTRIBUTING.md](./CONTRIBUTING.md) to learn how to get started contributing.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,119 @@
...
-For bug fixes that address open issues, the footer should be formatted like so:
+Provide the system information which is not limited to the below:
 
-```bash
-Closes #17, #26
-```
-and for Pull Requests, use the format:
+- Browser (if applies) [e.g. chrome, safari]
+- Version of jest-json-schema: [e.g. 5.0.0]
+- Node version:[e.g 10.15.1]
```
Comment (mtomcal) [inline CONTRIBUTING.md]: OS?

Comment (nellyk) [reply CONTRIBUTING.md]: updated f6a892b

**Pull request merged**